### ISSUE TITLE
The codebase needs a consistent identation

### ISSUE THREAD

**Issue by wdlkmpx** [emorozov/gtkballs#5]
I'm reading and there is a mix of tabs and spaces, and at least in Geany it's hard to follow the code blocks. It's very noticeable in Geany.

With C code, I found that spaces are often the best choice (3 or 4 spaces), I used to use only tabs in bash scripts, but in C I got used to spaces and parameters that take many lines, so that it's readable.

This is easy to do, and basically all the .c files could be adjusted to follow a consistent indentation.

**Comment by emorozov** [comment]
Yeah, the coding style of the current codebase is very inconsistent. I don't really know which code style is mainstream in C now, so I'm open to any choice as long as it is consistent.

**Comment by wdlkmpx** [comment]
When it comes C, my experience is that it must be spaces.

How many spaces? At first I thought 4, but since I'm using a small screen.. 3.

In the beginning I only used tabs and converted mixed indentation to tabs.. with Geany it's a matter of minutes.

In C you find a lot of functions that take many parameters, to make things readable, you split the call into several lines, the parameters go after the parenthesis.. this requires precision... spaces.

If you don't mind, I'll convert all the files to use only spaces (3) as the indentation. Then I'll perform more GTK updates.

This is a one time massive change so I can perform some updates leaving the codebase almost gtk3-ready (or maybe a working gtk3 port, depends on how much I learn), it doesn't matter if the coding style deviates or changes completely again..

**Comment by emorozov** [comment]
I'm used to indents being a power of 2. But I don't mind as long as it is consistent. Are you going to use an automatic tool?

I don't know what style is currently used for C projects. Of course I'd like to have something mainstream, so that the code doesn't look alien for people used to work on other projects.

On the other hand, you're the only person working on the project in 18 years. I've started some feeble attempt to rewrite just several days before, but stopped since you've suggested another approach. So I think ultimately it's your choice which style to use.

**Comment by wdlkmpx** [comment]
Ok, I'll perform a consistent indentation.

I'll probably be able to produce a fully working gtk3 port. It's
painful, but I learn something while updating a project and apply the
same knowledge and code to other projects, so I tend to do things
faster and faster.

I like small projects without too much gtk stuff, it's possible to
make them support several gtk generations, making the app universal
for old and new systems and devices..

Do you know other old gtk games that might need some attention?

**Comment by emorozov** [comment]
> Do you know other old gtk games that might need some attention?

Unfortunately, no. I mostly played X games that were implemented using plain xlib. And it was long time ago. Now I only play OpenXCom and OpenTTD occasionally. :)
